src/perf/parse-all-benchmark.ts

```diff
@@ -63,19 +63,51 @@ function padLeft(str: string, num: number): string {
   return str;
 }
 
+function memUsed(usage?: number) {
+  if (usage == null) {
+    usage = process.memoryUsage().rss;
+  }
+  return `${(usage / (1024 * 1024)).toFixed(1)}MiB`;
+}
+
 async function measure() {
+  global.gc();
+  const initialMemUse = process.memoryUsage().rss;
+  console.log(`Initial rss: ${memUsed(initialMemUse)}`);
   const start = now();
   let document: any;
   for (let i = 0; i < 10; i++) {
     document = await analyzer.analyze('ephemeral.html', fakeFileContents);
   }
 
+  global.gc();
+  const afterInitialAnalyses = process.memoryUsage().rss;
+
   const measurements = await analyzer.getTelemetryMeasurements();
   printMeasurements(measurements);
 
   console.log(`\n\n\n${document.getFeatures().size} total features resolved.`);
   console.log(
       `${((now() - start) / 1000).toFixed(2)} seconds total elapsed time`);
+
+  for (let i = 0; i < 100; i++) {
```

Comment:
It seems like we're testing two different things here: speed & memory leaks. Can we split into two different functions and add a way to test each individually?